rosidl_generator_dotnet/resource/msg.h.em

```diff
@@ -57,7 +57,9 @@ void @(msg_prefix)_CDECL @(msg_typename)__destroy_native_message(void *);
 
 @(msg_prefix)_EXPORT
 void @(msg_typename)__write_field_@(member.name)(void *, @(msg_type_to_c(member.type)));
-
+@[  else]@
```

Comment:
Super nitpicky, but just to line up the if/else words in the file:

```suggestion
@[    else]@
```

Reply:
No, Sean, it's perfect. It makes the code more readable